recipes-domd/agl/files/0001-Do-not-try-to-use-XDG-environment-while-setting-up.patch

```diff
@@ -0,0 +1,29 @@
+From 13abdd7af02592cf90f79b88b4ba3ebfa8fd9ee4 Mon Sep 17 00:00:00 2001
```

Comment:
I second that.

For now this looks like a huge copy-paste across projects. They will diverge in the future and we will have hard time trying to backport fixes to and fro.

Maybe it is worth to include those files as a separate layer from the original repository?

Reply:
Those come from different metal-layers AFAIK, so it might not be a good idea to include. What is more is that we do not have AGL based development product for daily use, we only integrate AGL on purpose. This is why, for example, CES demos have AGL integrated and prod-devel not. So, I am not convinced we should "include" AGL, but instead directly use it as it is done in this series.

Reply:
Yes, but how to cope with fixes that were done for one AGL-based product, but weren't propaged to other ones?